wdl/methods/phasing/PhysicalAndStatisticalPhasing.wdl

```diff
@@ -47,6 +47,13 @@ workflow PhysicalAndStatisticalPhasing {
         locus = region
     }
 
+    call FilterPhasingInfo as FilterPhasing { input:
```

Comment:
I would not call this operation "Filtering" as that implies hard/soft removal of entire records, whereas here we are just modifying genotypes. "Info" is also (ironically!) noninformative. `UnphaseGenotypes` would be more clear and succinct, IMO, but if you come up with something better, feel free to go with that!